command/job_run.go

```diff
@@ -345,6 +345,9 @@ func (c *JobRunCommand) Run(args []string) int {
 
 	evalID := resp.EvalID
 
+	// #13844: canonicalize the job in case it was a partial API definition
```

Comment:
Just FYI, there's no need to have the issue number in the comments. That's what `git blame` is for 😀